lang/en/block_qr.php

```diff
@@ -51,6 +51,7 @@
 $string['longitude'] = 'Longitude';
 $string['longitude_error'] = 'Only decimal values between -180 and 180 are permitted. Example: 10.49184';
 $string['medium'] = 'Medium';
+$string['nocontent'] = 'No content found';
```

Comment:
It would be nice, if the message would also contain a hint, how a content can be added.

```suggestion
$string['nocontent'] = 'No content has been configured yet. Click on the gear to configure a content for this QR block.';
```